rxjs-run/test/run.ts

```diff
@@ -0,0 +1,98 @@
+// tslint:disable-next-line
+import 'mocha';
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+import {run} from '../src/index';
+import {Observable, of, from, range} from 'rxjs';
+import {take, startWith, map, delay, concatMap, tap} from 'rxjs/operators';
+
+describe('run', function() {
+  it('should be a function', function() {
+    assert.strictEqual(typeof run, 'function');
+  });
+
+  it('should throw if first argument is not a function', function() {
+    assert.throws(() => {
+      (run as any)('not a function');
+    }, /First argument given to Cycle must be the 'main' function/i);
+  });
+
+  it('should throw if second argument is not an object', function() {
+    assert.throws(() => {
+      (run as any)(() => {}, 'not an object');
+    }, /Second argument given to Cycle must be an object with driver functions/i);
+  });
+
+  it('should throw if second argument is an empty object', function() {
+    assert.throws(() => {
+      (run as any)(() => {}, {});
+    }, /Second argument given to Cycle must be an object with at least one/i);
+  });
+
+  it('should return a dispose function', function() {
+    const sandbox = sinon.sandbox.create();
```

Comment:
Is deprecated, use `sinon.createSandbox()` instead

Reply:
Hmm, the types don't allow me to write sinon.createSandbox(). Maybe you updated sinon's version in the pnpm PR? In that case we can do a merge of those two branches, whichever gets merged second adapts accordingly.

Reply:
Yeah, I updated all dependencies